web/html/src/manager/state/recurring-states-list.tsx

```diff
@@ -60,6 +60,18 @@ class RecurringStatesList extends React.Component<Props, State> {
         title={t("Recurring States")}
         icon="spacewalk-icon-salt"
         buttons={this.props.disableCreate ? [] : buttons}
+        summary={[
```

Comment:
Perhaps we could use a fragment here instead of an array, a-la `summary={<>...</>}`?

Reply:
Sure Done